lib/screens/sentence_item.dart

```diff
@@ -8,20 +8,36 @@ class SentenceItem extends StatelessWidget {
 
   SentenceItem(this._phrase, this._index);
 
+  Color _tileBackgroundColor(BuildContext context) {
+    if (_index % 2 == 0) return Theme.of(context).primaryColorLight;
+    return Color.fromRGBO(1, 1, 1, 0.0);
```

Comment:
is it possible to express these without magic numbers?

Reply:
Will remove magic number later. Wanted something quick for the moment